src/scss/Navigation.scss

```diff
@@ -18,6 +18,7 @@ $pillborder: #ebecee;
 
   @media (max-width: 500px) {
     margin: 6px auto;
+    overflow-x: hidden;
```

Comment:
Hmmmmm... this is independently OK, but are we sure it won't cause inputs to scroll off?